src/components/Charts/MetricChart.tsx

```diff
@@ -109,14 +109,25 @@ function getVaccinationSeries(projection: Projection): Series[] {
     {
       type: SeriesType.LINE,
       data: filterNull(projection.getDataset('vaccinationsCompleted')),
-      label: 'Vaccinations Completed',
```

Comment:
I'm not sure if we want to change the meaning of "full vaccinated" yet, so we should probably get this confirmed

Reply:
FWIW- The latest copy I've seen is "2+ doses or J&J" but yeah, I'm not sure we should change this until we change the rest of the site to match.